getting-started/data-node-authentication.md

```diff
@@ -11,90 +11,181 @@ and the access node need to have authentication set up for those
+database and use the same database on all the data nodes, so you need
+to ensure that a user that is connected to he access node can access
+the same databases on the data nodes.
 
->:WARNING: Setting up a secure system is a complex task and this section
-should not be read as recommending any particular security measures
-for securing your system. The chapter contains technical
-instructions for how to set up the different authentication methods.
+>:WARNING: Setting up a secure system is a complex task and this section should not be read as recommending any particular security measures for securing your system. The chapter contains technical instructions for how to set up the different authentication methods.
 
 ## Setting up Password Authentication [](distdb-auth-pass)
 
-The easiest authentication to set up is password
+The easiest authentication method to set up is password
 authentication. Configuring password authentication between the access
 node and data nodes is a two step process:
 
 1. Each data node needs to be configured to accept and authenticate
    connections using passwords.
 
 2. The access node needs data node passwords for each role using the
-   access node. We will deal with each step separately below.
+   access node.
 
 In this section, the focus is on setting up SCRAM SHA-256 password
 authentication. For other password authentication methods, [see the
 PostgreSQL manual][auth-password].
 
->:TIP: The `password` method should not be used since it sends the
-> password in cleartext, and the `md5` is using a cryptographic hash
-> function that is no longer considered secure.
+Let's assume that you want to add a user `new_user` to both the access
+node and data node and that the user should have the password `xyzzy`.
+
+>:TIP: This description is using `scram-sha-256` since the `password`
+> method is not recommended (it sends the password in cleartext), and
+> the `md5` method is using a cryptographic hash function that is no
+> longer considered secure.
 
 ### Setting up Password Authentication on the Data Node [](distdb-auth-pass-data-node)
 
-In order to set up password authentication for a user `new_user`, it
-is necessary to update the HBA (Host-Based Authentication) file on the
-data nodes to enable password authentication for `new_user`. The
-default name of the HBA file is `pg_hba.conf`. Details of [the
-`pg_hba.conf` format can be found in the PostgreSQL
+In order to set up password authentication for `new_user` using
+password `xyzzy`, you need to:
+
+1. Set the correct password encryption in the PostgreSQL configuration
+   file.
+2. Check that the correct HBA (Host-Based Authentication) file is in
+   the PostgreSQL configuration file.
+3. Update the HBA file on the data nodes to enable password
+   authentication.
+4. Reload the server to read the new configuration.
+5. Add the user on the data node with the right password.
+
+### Set up password encryption method
```

Comment:
I feel that this is a subsection of the `Setting up Password Authentication on the Data Node` section and should get one more level of hierarchy here (so `####`).  Same comment for the next three headers.

Reply:
Right, that was the intention. Good catch.